src/modules/cmdpal/Exts/ProcessMonitorExtension/TerminateProcess.cs

```diff
@@ -10,10 +10,12 @@ namespace ProcessMonitorExtension;
 internal sealed partial class TerminateProcess : InvokableCommand
 {
     private readonly ProcessItem _process;
+    private readonly ProcessListPage _owner;
 
-    public TerminateProcess(ProcessItem process)
+    public TerminateProcess(ProcessItem process, ProcessListPage owner)
     {
         _process = process;
+        _owner = owner;
         Icon = new("\ue74d");
         Name = "End task";
```

Comment:
ok not that line, but the "End task" one lol -- idk why that happened

Reply:
Yea, pretty much all the extensions beyond the couple from PT need a whole pass to stand up localization for them. Better not to muddle this PR with that change. 

Honestly I should have reverted this one instead of even including it as a drive-by, but 🤷